myftp.py

```diff
@@ -9,14 +9,19 @@ def initConnection():
   hostName = sys.argv[1]
   if (hostName == 'test'):
     hostName = ftputils.TEST_HOST
-  ftpController = FTPController(hostName)
-  print(ftpController.connect())
-  return ftpController
+    ftpController = FTPController(hostName)
+    print(ftpController.connect())
+    return ftpController
+  else:
+    print('Invalid host name specified, exiting.')
+    return None
 
 def promptLogin(ftpController):
   username = input('Please enter your username: ')
   password = input('Please enter your password: ')
-  print(ftpController.login(username, password))
+  loginStatusCode = ftpController.login(username, password)
```

Comment:
Perhaps we can do this internally within the FTPController's login() method and perform a `sys.exit()`

Reply:
no